code/addons/interactions/package.json

```diff
@@ -54,28 +54,21 @@
   },
   "dependencies": {
     "@storybook/global": "^5.0.0",
-    "@storybook/instrumenter": "workspace:*",
-    "@storybook/test": "workspace:*",
```

Comment:
Was the test package mistakenly removed from the list?

Reply:
`@storybook/instumenter` was removed when I consolidated it, but then I unconsolidated it based on @kasperpeulen's suggestion, but forgot to put it back, good catch!

`@storybook/test` isn't needed, it's just used in stories.